core/storage/user/gae_models_test.py

```diff
@@ -611,6 +751,61 @@ def test_export_data_multiple_explorations(self):
 
 
 class CollectionProgressModelTests(test_utils.GenericTestBase):
+    """Tests for CollectionProgressModel."""
+    USER_ID_1 = 'user_id_1'
+    USER_ID_2 = 'user_id_2'
+    USER_ID_3 = 'user_id_3'
+    COLLECTION_ID_1 = 'col_id_1'
+    COLLECTION_ID_2 = 'col_id_2'
+    COMPLETED_EXPLORATIONS_1 = ['exp_1', 'exp_2', 'exp_3']
```

Comment:
Here and below, be careful to distinguish between "exploration" and "exploration ID" in the naming -- I think these are IDs.

Reply:
Ack